js/index.js

```diff
@@ -0,0 +1,134 @@
+            alert(response.status + " " + response.statusText);
+        }
+    }).catch(function() {
+        alert("fetch error!");
+    });
+};
+
+const deleteUser = id => {
+    const response = fetch(requestUrl + "/" + id, {
+        method: 'DELETE'
+    }).then(getUsers).catch(function() {
+        alert("fetch error!");
+    });
+};
+
+const userPost = function() {
+    let date = new Date();
+    let Id = document.getElementById("inId").value;
+    let Name = document.getElementById("inName").value;
+    let Avatar = document.getElementById("inLink").value;
+    const response = fetch(requestUrl, {
+        method: 'POST',
+        body: JSON.stringify({id: Id, name: Name, avatar: Avatar, createdAt: date.toISOString()})
+    }).then(function(response) { 
+        if (response.ok) {
+            getUsers();
+        }
+        else {
+            alert(response.status + " " + response.statusText);
+        }
+    }).catch(function() {
+        alert("fetch error!");
+    });
+}
+
+const userPut = function() {
+    let date = new Date();
+    let Id = document.getElementById("inId").value;
+    let Name = document.getElementById("inName").value;
+    let Avatar = document.getElementById("inLink").value;
+    const response = fetch(requestUrl + "/" + Id, {
+        method: 'PUT',
+        body: JSON.stringify({id: Id, name: Name, avatar: Avatar, createdAt: date.toISOString()})
+    }).then(function(response) { 
+        if (response.ok) {
+            getUsers();
+        }
+        else {
+            alert(response.status + " " + response.statusText);
+        }
+    }).catch(function() {
+        alert("fetch error!");
+    });
+}
+
+const userGet = id => {
+    fetch(requestUrl + "/" + id).then(function(response) {
+        if (response.ok) {
+            response.json().then(function(json) {
+                tableUsers.innerHTML = "";
+                tableUsers.innerHTML += createTemplate(json);
+                document.getElementById("inId").value = json.id;
+                document.getElementById("inName").value = json.name;
+                document.getElementById("inLink").value = json.avatar;
+            })
+        }
+        else {
+            alert(response.status + " " + response.statusText);
+        }
+    }).catch(function() {
+        alert("fetch error!");
+    });
+};
+
+const createTemplate = data => {
+    let template = "<tbody>";
+    let i = 0;
+    if (data.length === undefined) {
+        template += `
+        <tr id="row${data.id}">
+            <td id="id" class="text-center">${data.id}</td>
+            <td id="pic">
+                <img class="rounded" src="${data.avatar}">
+            </td>
+            <td id="name" class="text-center">${data.name}</td>
+            <td>
+                <button class="btn btn-info" onclick="deleteUser(${data.id})">Delete</button>
+            </td>
+            <td>
+                <button class="btn btn-info" onclick="userGet(${data.id})">Get by id</button>
```

Comment:
This makes no sense. We are about to see it on page with 1 user. and this button navigates to this page again. This button could be removed